.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -60,57 +59,53 @@ body:
 
         Please include screenshots (you can drag any image here) where applicable. Redact any sensitive information.
     validations:
-      required: true
+      required: false
   - type: textarea
     attributes:
       label: "System information and logs"
       description: |
-        Provide details about the environment you experienced the issue in. Change msentraid for the provider you are using:
-      value: |
-        ### Environment
-        * broker version: please run `snap info authd-msentraid`
-        * authd version: please run `/usr/libexec/authd version`
-        * gnome shell version: please run `apt policy gnome-shell`
-        * Distribution: (**NAME** in `/etc/os-release`)
-        * Distribution version: (**VERSION_ID** on `/etc/os-release`):
-
-        ### Log files
-        Please redact/remove sensitive information:
-
-        #### Authd entries:
-
-        ```raw
-        journalctl -u authd.service
-        ```
-
-        ####  MS Entra ID broker entries:
-
-        ```raw
-        journalctl -u snap.authd-msentraid.authd-msentraid.service
-        ```
-
-        ### Application settings
-
-        Please redact/remove sensitive information:
-
-        ####  Broker configuration:
-
-        ```raw
-        cat /var/snap/authd-msentraid/current/broker.conf
+        Copy this command and run it in a terminal. It will open a text editor with the system information and logs. Paste the output below. Redact any sensitive information from the logs.
+        ```bash
+        gnome-text-editor --standalone - <<EOF
+        #### authd version
+        \`\`\`
+        $(/usr/libexec/authd version)
+        \`\`\`
+
+        #### authd-msentraid broker version
+        \`\`\`
+        $(snap info authd-msentraid)
+        \`\`\`
+
+        #### gnome-shell version
+        \`\`\`
+        $(apt policy gnome-shell)
+        \`\`\`
+
+        #### Distribution
+        \`\`\`
+        $(lsb_release -a)
+        \`\`\`
+
+        #### Logs
+        \`\`\`
+        $(journalctl -o short-monotonic --lines 500  _SYSTEMD_UNIT=authd.service \+ UNIT=authd.service \+ _SYSTEMD_UNIT=snap.authd-msentraid.authd-msentraid.service \+ UNIT=snap.authd-msentraid.authd-msentraid.service \+ '_CMDLINE="gdm-session-worker [pam/gdm-authd]"' | sed -E 's/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}/<UUID redacted>/g')
```

Comment:
Wondering if we should instead ship a script to get all these debugging infos, parsed and sanitized that can be called.

Instead of using all the command lines, though, just rely on binaries so that we don't have to check for all the gdm tools (that may be relevant for some issues), such as `/usr/bin/gnome-shell /usr/libexec/gdm-* /usr/sbin/gdm3`

Reply:
> Wondering if we should instead ship a script to get all these debugging infos, parsed and sanitized that can be called.

Yes, we discussed that today. We do want to consider shipping a script (or even building the functionality to produce this output into the `authd` binary) but for now we want to go with the copy-pastable command.

> Instead of using all the command lines, though, just rely on binaries so that we don't have to check for all the gdm tools (that may be relevant for some issues), such as /usr/bin/gnome-shell /usr/libexec/gdm-* /usr/sbin/gdm3

Do we really need all output from gnome-shell and gdm though? The `_CMDLINE="gdm-session-worker [pam/gdm-authd]"` for example only matches those gdm lines which are related to gdm-authd. Are there any lines missing if you run that command?

Reply:
I'm fine with that, but we may still want the info from the password service in case it runs or we fallback to the local broker.

So maybe just use `/usr/libexec/gdm-* and /usr/bin/gnome-shell` limiting the output to the wanted `-n` lines.

Reply:
> So maybe just use /usr/libexec/gdm-* and /usr/bin/gnome-shell

The match arguments to journalctl don't support wildcards, so we need to list all executables we want to match. Can you specify those? Is it all of these?
```bash
$ ls /usr/libexec/gdm-*
/usr/libexec/gdm-auth-config-debian
/usr/libexec/gdm-host-chooser
/usr/libexec/gdm-runtime-config
/usr/libexec/gdm-session-worker
/usr/libexec/gdm-simple-chooser
/usr/libexec/gdm-wayland-session
/usr/libexec/gdm-x-session
```

The concerns I mentioned in https://github.com/ubuntu/authd/pull/484#issuecomment-2302710155 remain, but I trust your judgment that we really need those logs.

Reply:
Area you sure? I get all the logs myself with `journalctl /usr/bin/gnome-shell /usr/libexec/gdm-* /usr/sbin/gdm3 -n 200`

Reply:
e.g. it's not journalctl supporting wildcards, but the shell does so when you call it without any quote, it will just include all the paths we care about.

Reply:
When combined with the other matches it doesn't seem to work as expected. This command

```
journalctl -o short-monotonic --lines 500  _SYSTEMD_UNIT=authd.service \+ UNIT=authd.service \+ _SYSTEMD_UNIT=snap.authd-msentraid.authd-msentraid.service \+ UNIT=snap.authd-msentraid.authd-msentraid.service \+ '_CMDLINE="gdm-session-worker [pam/gdm-authd]"' \+ /usr/bin/gnome-shell \+ /usr/libexec/gdm-*
```

doesn't list any of the gdm entries.

Reply:
Even without the other matches it doesn't work for me. The command you use doesn't return any entries on my system:

```bash
$ journalctl /usr/bin/gnome-shell /usr/libexec/gdm-* /usr/sbin/gdm3 -n 200
-- No entries --
```

Reply:
It doesn't return anything for me either